sklearn/covariance/tests/test_robust_covariance.py

```diff
@@ -100,8 +101,15 @@ def test_outlier_detection():
     assert_raises(NotFittedError, clf.decision_function, X)
     clf.fit(X)
     y_pred = clf.predict(X)
-    decision = clf.decision_function(X, raw_values=True)
-    decision_transformed = clf.decision_function(X, raw_values=False)
+
+    assert_warns(DeprecationWarning,
+                 clf.decision_function, X, raw_values=True)
```

Comment:
can you check that the return value of this is the same as the ``decision`` value below?